test cases/unit/97 link full name/proguser/receiver.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+int  __attribute__((weak)) get_checked(void) {
+    return -1;
+}
+
+
+#define CHECK_VALUE (100)
+#define TEST_SUCCESS (0)
+#define TEST_FAILTURE (-1)
+
+int main() {
```

Comment:
This needs to be changed to `int main(void)`, otherwise one of our CIs will fail (combination of all the warnings and `-Werror`)

Reply:
Fixed in commit 90a5f0ad1077eb7ea26af464c5d8dfc5f3724fe6